docs/backend-introduction.md

```diff
@@ -65,19 +65,19 @@ We would suggest to use __3 tabs in the shell__:
 
 ### Relationships in Backend
 
-First of all, you have to have a `Group` Model, allowing to create objects like "Foodsavers Berlin". One Group usually has many `Stores`, like "Bakery Smith". Each store can define events where foodsavers can come by and save food. These events are called `PickupDate` (one time event) or `PickupDateSeries` (repetitive event).
+First of all, you have to have a `Group` Model, allowing to create objects like "Foodsavers Berlin". One Group usually has many `Stores`, like "Bakery Smith". Each store can define events where foodsavers can come by and save food. These events are called `ActivityDate` (one time event) or `ActivityDateSeries` (repetitive event).
```

Comment:
I wonder if it should be called `Activity` instead of `ActivityDate`? Similar with `ActivityDateSeries`.

Reply:
I reverted the whole backend introduction as this PR doesn't touch the backend.